website/docs/cloud-docs/policy-enforcement/prewritten-sentinel.mdx

```diff
@@ -39,26 +36,19 @@ To create new policy sets and policies, your HCP Terraform or Terraform Enterpri
 
 You must have a GitHub account connected to HCP Terraform or Terraform Enterprise to manually connect policy sets to your workspaces. Refer to [Connecting VCS Providers](/terraform/cloud-docs/vcs) for instructions.
 
-## Obtain policies
+## Get policies
 
-You can use the policy libraries created and maintained by HashiCorp. The libraries are stored in the following GitHub repositories:
+Refer to [Pre-written policy library reference](/terraform/cloud-docs/policy-enforcement/prewritten-library) for a complete list of available policies. You can also [browse the registry](https://registry.terraform.io/browse/policies) to discover additional policy libraries.
 
-- [policy-library-cis-aws-efs-terraform](https://github.com/hashicorp/policy-library-cis-aws-efs-terraform)
-- [policy-library-cis-aws-rds-terraform](https://github.com/hashicorp/policy-library-cis-aws-rds-terraform)
-- [policy-library-cis-aws-vpc-terraform](https://github.com/hashicorp/policy-library-cis-aws-vpc-terraform)
-- [policy-library-cis-aws-iam-terraform](https://github.com/hashicorp/policy-library-cis-aws-iam-terraform)
-- [policy-library-cis-aws-s3-terraform](https://github.com/hashicorp/policy-library-cis-aws-s3-terraform)
-- [policv-library-cis-aws-cloudtrail-terraform](https://github.com/hashicorp/policy-library-cis-aws-cloudtrail-terraform)
-- [policy-library-cis-aws-kms-terraform](https://github.com/hashicorp/policy-library-cis-aws-kms-terraform)
-- [policy-library-cis-aws-ec2-terraform](https://github.com/hashicorp/policy-library-cis-aws-ec2-terraform)
-
-Use one of the following methods to obtain pre-written policies:
+Use one of the following methods to get pre-written policies:
 
 - **Download policies from the registry**: Use this method if you want to assemble custom policy sets without customizing policies.  
 - **Fork the HashiCorp policy GitHub repository**: Use this method if you intend to customize the policies. 
-- **Add the Terraform module to your configuration**: Use this method to implement specific versions of the policies as-is. This method also connects the policies to workspaces in the Terraform configuration file instead of connecting them as a separate step. 
+- **Add the Terraform module to your configuration**: Use this method to implement specific versions of the policies as-is. This method also connects the policies to workspaces in the Terraform configuration file instead of connecting them as a separate step.
+
+<Tabs>
```

Comment:
is there a reason why we're opting for tabs rather than headers? i think headers make it easier to "ctrl+f" and link to specific methods

Reply:
I thought this would be cleaner. The control+f principle is more for reference pages. We expect people to more or less follow a usage page beginning to end. Besides, control+f still finds "module, for example, if you are on the "Download from registry" tab.